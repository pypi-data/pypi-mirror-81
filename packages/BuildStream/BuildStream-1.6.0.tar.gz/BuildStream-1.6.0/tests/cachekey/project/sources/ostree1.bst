kind: import
sources:
- kind: ostree
  url: https://example.com/repo
  ref: ccc885b96749f9d1774c7fa0c3262a9a3694e2d2643d1f8d420f5d23adf5db48
  track: testing/x86_64
