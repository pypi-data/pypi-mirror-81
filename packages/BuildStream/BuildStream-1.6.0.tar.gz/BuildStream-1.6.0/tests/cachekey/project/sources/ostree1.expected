8bccdd60fc3a89eb9ed807dd31e868ea48c98411c564ca4aefc354b9956675de